<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Your Profile</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #808080;
color: white;
font-family: Comic sans MS, sans-serif;
font-size: 14px;
font-weight: 400;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item row="0" column="1">
     <widget class="QFrame" name="verticalFrame_5">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #A7A7A7;
border-radius: 8px;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Личная информация:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QFrame" name="verticalFrame">
           <property name="styleSheet">
            <string notr="true">border: 2px solid #A7A7A7;
border-radius: 8px;</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLineEdit" name="surname_line">
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Введите фамилию...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="name_line">
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Введите имя...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="patronymic_line">
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Введите отчество...</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLineEdit" name="email_line">
                <property name="text">
                 <string/>
                </property>
                <property name="placeholderText">
                 <string>Введите электронную почту...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="telephone_number_line">
                <property name="text">
                 <string/>
                </property>
                <property name="placeholderText">
                 <string>Введите номер телефона...</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QLineEdit" name="city_line">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="placeholderText">
                   <string>Введите город...</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTextEdit" name="information_line">
                  <property name="html">
                   <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Comic sans MS','sans-serif'; font-size:14px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                  <property name="placeholderText">
                   <string>Введите информацию о себе...</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <widget class="QPushButton" name="update_data_btn">
                    <property name="styleSheet">
                     <string notr="true">QPushButton {
	border: 2px solid #A7A7A7;
	border-radius: 8px;
 }

QPushButton::hover {
	background-color: #8E8E8E;
 }

QPushButton::pressed {
	background-color: #5F655F;
 }</string>
                    </property>
                    <property name="text">
                     <string>Применить</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QLabel" name="notification_data">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Пароль:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="verticalFrame_7">
         <property name="styleSheet">
          <string notr="true">border: 2px solid #A7A7A7;
border-radius: 8px;</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLineEdit" name="reset_passw">
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
              <property name="placeholderText">
               <string>Введите новый пароль...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="reset_passw2">
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
              <property name="placeholderText">
               <string>Повторите пароль...</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="reset_passw_btn">
                <property name="styleSheet">
                 <string notr="true">QPushButton {
	border: 2px solid #A7A7A7;
	border-radius: 8px;
 }

QPushButton::hover {
	background-color: #8E8E8E;
 }

QPushButton::pressed {
	background-color: #5F655F;
 }</string>
                </property>
                <property name="text">
                 <string>Применить</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="notification_passw">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="verticalFrame_8">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #A7A7A7;
border-radius: 8px;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="greetings">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Здравствуйте, username!</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ava">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Аватарка</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="update">
         <property name="text">
          <string>Обновить профиль</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="logout">
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border: 2px solid #A7A7A7;
	border-radius: 8px;
 }

QPushButton::hover {
	background-color: #8E8E8E;
 }

QPushButton::pressed {
	background-color: #5F655F;
 }</string>
         </property>
         <property name="text">
          <string>Выйти...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
